src/goose.rs

```diff
@@ -302,8 +306,18 @@ static APP_USER_AGENT: &str = concat!(env!("CARGO_PKG_NAME"), "/", env!("CARGO_P
 #[macro_export]
```

Comment:
This doc string probably wants changing to something like

    task!(foo) conceptually expands to GooseTask::new(foo), but also does some boxing, to work around a limitation in the compiler.